/**:
  controller_manager:
    ros__parameters:
      update_rate: 100
      use_sim_time: false

      joint_state_broadcaster:
        type: joint_state_broadcaster/JointStateBroadcaster

      joint_trajectory_left:
        type: joint_trajectory_command_broadcaster/JointTrajectoryCommandBroadcaster

      joint_trajectory_right:
        type: joint_trajectory_command_broadcaster/JointTrajectoryCommandBroadcaster

      joint_trajectory_left_hand:
        type: joint_trajectory_command_broadcaster/JointTrajectoryCommandBroadcaster

      joint_trajectory_right_hand:
        type: joint_trajectory_command_broadcaster/JointTrajectoryCommandBroadcaster

  joint_trajectory_left:
    ros__parameters:
      joints:
        - arm_l_joint1
        - arm_l_joint2
        - arm_l_joint3
        - arm_l_joint4
        - arm_l_joint5
        - arm_l_joint6
        - arm_l_joint7
      interfaces:
        - position
      action_monitor_rate: 20.0
      use_local_topics: true

  joint_trajectory_right:
    ros__parameters:
      joints:
        - arm_r_joint1
        - arm_r_joint2
        - arm_r_joint3
        - arm_r_joint4
        - arm_r_joint5
        - arm_r_joint6
        - arm_r_joint7
      interfaces:
        - position
      action_monitor_rate: 20.0
      use_local_topics: true

  joint_trajectory_left_hand:
    ros__parameters:
      joints:
      - left_thumb_1_joint
      - left_thumb_2_joint
      - left_index_1_joint
      - left_middle_1_joint
      - left_ring_1_joint
      - left_little_1_joint

      interfaces:
        - position

      action_monitor_rate: 20.0
      use_local_topics: true

  joint_trajectory_right_hand:
    ros__parameters:
      joints:
      - right_thumb_1_joint
      - right_thumb_2_joint
      - right_index_1_joint
      - right_middle_1_joint
      - right_ring_1_joint
      - right_little_1_joint

      interfaces:
        - position

      action_monitor_rate: 20.0
      use_local_topics: true
